### ISSUE TITLE
Map Filter Panel - 1.4.3 Contrast Minimum - 1 contrast error present on the text of the unselected Basemap Panel option

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9903]
As a user with visual impairments, I expect all contrast between text and background to be at least 4.5:1, to ensure I can discern text.

**Linked Work Item:**
#9493 

**Expected Result:**
 A contrast ratio of 4.5:1 should be present between all text and background colours to ensure good visibility for people with vision impairments.

**Actual Result:**
A contrast ratio of 2.67:1 is present on the text of the option which is not selected (grey-on-white).

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/e57f9b2d-2882-40e6-93ce-098a9eac8509)

**Comment by johnatawnclementawn** [comment]
@imogenroberts please pull latest and RETEST

**Comment by imogenroberts** [comment]
RETEST: No contrast issues are present anywhere on the page.

**Evidence:**
![Map Filter - 1 4 3 Contrast FIXED](https://github.com/archesproject/arches/assets/77161085/a962de4f-e188-41be-841c-ddf29e77409e)
